packages/component/README.md

```diff
@@ -0,0 +1,20 @@
+# @firebase/component
+
+_NOTE: This is specifically tailored for Firebase JS SDK usage, if you are not a
+member of the Firebase team, please avoid using this package_
+
+## Installation
```

Comment:
Should we get rid of the rest of the instructions from here on down if nobody should be installing it anyway?

Reply:
Good point, but I think it's harmless to keep it, and it's consistent with other packages, e.g. `@firebase/util`.